packages/react-ui/src/widgets/legend/Note.js

```diff
@@ -19,9 +16,9 @@ export default function Note({ children }) {
   return (
     <Box mt={1} data-testid='note-legend'>
       <Typography variant='caption'>Note:</Typography>{' '}
-      <Typography className={classes.note} variant='caption'>
```

Comment:
This is no longer necessary, as the `caption` variant has a regular weight by default.

You can use just `<Typography variant='caption' /> `and remove the customization.